sapp/ui/issues.py

```diff
@@ -48,6 +49,18 @@
 SinkKindText = aliased(SharedText)
 
 
+class WarningMessageQueryType(graphene.ObjectType):
+    message = graphene.String()
+    code = graphene.Int()
+
+def get_warning_message(
+    session: Session, code: int,) -> List[WarningMessage]:
+    return (
+        session.query(WarningMessage)
+        .filter(WarningMessage.code == code)
+        .all()
+    )
```

Comment:
Would we be able to use `Query.one_or_none()` instead, since we expect [`WarningMessage.code` to be unique](https://github.com/facebook/sapp/blob/main/sapp/models.py#L1248
) in `warning_messages` table?


```suggestion
def get_warning_message(
    session: Session, code: int,) -> WarningMessage:
    return (
        session.query(WarningMessage)
        .filter(WarningMessage.code == code)
        .one_or_none()
    )
```

Reply:
Resolved